### ISSUE TITLE
including thredded stylesheets with @include "thredded" in production causes assets pipeline to fail

### ISSUE THREAD

**Issue by pyrabbit** [thredded/thredded#575]
Hi everyone. We are interested in using thredded but it is wreaking havoc across my asset pipeline when running rails in production mode.

No errors are generated and thredded seems to prevent my custom application stylesheets from being compiled.

I am including thredded in a `custom_thredded.scss` file by placing `@import "thredded";` or `@import "thredded/thredded";` at the type of my file will cause an unknown failure during the compilation process when running in production. Commenting out the import line will allow my assets to be compiled again but I will ultimately be missing all the thredded stylesheets. 

I am running Rails 5 and Thredded 0.11. Has anyone else experienced this issue or a similar issue and was able to resolve it?

**Comment by jayroh** [comment]
Could you post a stack trace of this when you run the asset compilation rake task (with RAILS_ENV=production)?

**Comment by pyrabbit** [comment]
@jayroh I ran `RAILS_ENV=production rake assets:precompile` on my dev machine and it completes with no errors. I set `config.assets.compile = false` in the `config/environments/development.rb` to force use of the static assets and I am seeing the same issues we see in production.

I then run `RAILS_ENV=production rails s` on my development machine and I am able to reproduce the same issue I am seeing on my live production servers. Beautiful thredded forums with none of my original application stylings.
